docs/sources/administration/correlations/add-permissions-to-create-new-correlations/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Add permissions to create new correlations
+menuTitle: Add permissions
+weight: 30
+---
+
+# Add permissions to create new correlations
+
+## Before you begin
+
+Adding access to create correlations for [Viewers and Editors]({{< relref "/docs/grafana/latest/administration/roles-and-permissions/" >}}) is available with [Role-based access control]({{< relref "/docs/grafana/latest/administration/roles-and-permissions/access-control/" >}}).
+
+## Add permissions to create correlations
+
+1. Got to the Administration section in Grafana
```

Comment:
Go*

Reply:
Fixed in f07528fedacad24fccf1694e79906b971c91ede8.